README.md

```diff
@@ -13,23 +13,21 @@ A Quick Overview of How to Use this Repository
 ----------------------------------------------
 
 To add your project to our build farm, fork this repository, modify the .yaml files, and give us a pull request.
-For Fuerte and older the ROS packages are maintained in the subfolder ``releases`` and ``doc``.
-For Groovy and newer the ROS packages are maintained in the subfolder named after the ROS distribution.
-Mappings for dependencies are maintained in the ``rosdep`` subfolder.
+Mappings for dependencies are maintained in the `rosdep` subfolder of https://github.com/ros/rosdistro
```

Comment:
do we know why there was double backticks everywhere before? maybe the linters will fail if we remove them?

Reply:
> do we know why there was double backticks everywhere before? maybe the linters will fail if we remove them?

I am not sure what "the linters" are in this context. There doesn't appear to be anything in travis-ci that runs over our markdown. Double backticks with spaces separating the backticks from the contents are used to create code blocks that can contain literal backticks but none of these need it here.